src/classes/CW_AudienceGroupTriggerHandlerTest.cls

```diff
@@ -0,0 +1,236 @@
+/**
+* File Name   :    CW_AudienceGroupTriggerHandlerTest.cls
+* Description :    D-106803o
+*                 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 Alex  Panka		Initial
+ */
+ 
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.